operations/sop.md

```diff
@@ -80,55 +80,77 @@ The mission is officially over at this point, and the conversation is open for a
 
 ## Composition
 
-Each mission must have an operation lead, this is usually the squad leader, but it does not have to be.
+Each mission must have an operation lead.
 
-The operation lead is responsible for dividing up members into squads.
+The operation lead is responsible for dividing up members into elements.
 
 ### Squad Composition
 
-The maximum squad size is 8, and should be no smaller than 4. Each squad is made up of 2 man teams.
+For the overwhelming number of contracts all contractors will be in one squad. A squad is broken into multiple elements.
+
+#### Roles
+
+**Operation Lead**  
+Responsible for setting the objectives for the squad. Responsible for keeping track of each element's progress and coordinating multi-element maneuvers.
+
+**Element Leader**
+Responsible for planning and executing their elements movements as part of the OL's set objectives.
+
```

Comment:
Add Team Leader here.. define their roles as executing the EL's orders as well as micro tactics for their team.

Reply:
I purposely left team leader out. Instead calling them primaries. The teams should only ever be 3 people maximum. I wanted to avoid the TL terminology since they shouldn't really be leading anything per se. Instead just doing the micro tactics.

Reply:
I think it's still important to stay consistent across the board IMHO. And even with a small team (3 members max), there should be a clear designee as the leader because they will be the one to lead the team in employment of their micro-tactics.